test/observability/helpers/generator.go

```diff
@@ -0,0 +1,42 @@
+package helpers
```

Comment:
Consider reviewing the azure_test.go which is the pattern I applied to the other outputs which makes use of a matcher that takes `Element`.  You may find this file along with some setup code in the actual test is not required

Reply:
I've refactored the loki tests accordingly.